app/retail/views.py

```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
     reasons = [
         {
             'title': 'Hackathon',
-            'img': static('v2/images/tribes/landing/hackathon.svg'),
+            'img': static('v2/images/bulb.svg'),
             'info': 'See meaningful projects come to life on your dapp'
         },
         {
             'title': 'Suggest Bounty',
-            'img': static('v2/images/tribes/landing/suggest.svg'),
+            'img': static('v2/images/pencil.svg'),
             'info': 'Get bottoms up ideas from passionate contributors'
         },
         {
-            'title': 'Grow Tribe',
-            'img': static('v2/images/tribes/landing/grow.svg'),
-            'info': 'Work seamlessly with your core contributors'
+            'title': 'Quadratic Funding',
+            'img': static('v2/images/funding.svg'),
+            'info': 'Incentivize your tribe with mini quadratic funding'
         },
         {
             'title': 'Workshops',
-            'img': static('v2/images/tribes/landing/workshop.svg'),
+            'img': static('v2/images/workshops.svg'),
             'info': 'Host workshops and learn together'
         },
         {
             'title': 'Chat',
-            'img': static('v2/images/tribes/landing/chat.svg'),
+            'img': static('v2/images/Chat.svg'),
             'info': 'Direct connection to your trusted tribe'
         },
         {
             'title': 'Town Square',
-            'img': static('v2/images/tribes/landing/townsquare.svg'),
-            'info': 'Broadcast your priorities and engagey our tribe'
+            'img': static('v2/images/townsquare.svg'),
+            'info': 'Engage the members of your tribe'
         },
         {
             'title': 'Payout/Fund',
-            'img': static('v2/images/tribes/landing/payout.svg'),
+            'img': static('v2/images/payout.svg'),
             'info': 'Easily co-manage hackathons with your team'
         },
         {
             'title': 'Stats Report',
-            'img': static('v2/images/tribes/landing/stats.svg'),
+            'img': static('v2/images/report.svg'),
             'info': 'See how your hackathons are performing'
+        },
+        {
+            'title': 'Kudos',
+            'img': static('v2/images/kudos.svg'),
+            'info': 'Show appreciation to your tribe members'
+        }
+    ]
```

Comment:
^ why are these files added again ?
Let's reuse what we have + remove duplicate svgs and let's keep them under `'v2/images/tribes/landing`
to keep it a better organized